src/styles/variables/colors.scss

```diff
@@ -39,13 +40,34 @@ $neutral-colors: (
 }
 
 $color-types: (
-    primary: os-color(blue),
-    secondary: os-color(orange),
-    tertiary: os-color(yellow),
-    success: os-color(green),
-    info: os-color(cyan),
-    warning: os-color(yellow),
-    danger: os-color(red)
+    primary: (
+        color: os-color(blue),
+        overlay: neutral-color(lightest)
+    ),
+    secondary: (
+        color: os-color(orange),
+        overlay: neutral-color(lightest)
+    ),
+    tertiary: (
+        color: os-color(yellow),
+        overlay: neutral-color(lightest)
+    ),
+    success: (
+        color: os-color(green),
+        overlay: neutral-color(lightest)
+    ),
+    info: (
+        color: os-color(cyan),
+        overlay: neutral-color(lightest)
+    ),
+    warning: (
+        color: os-color(yellow),
+        overlay: neutral-color(darkest)
+    ),
+    danger: (
+        color: os-color(red),
+        overlay: neutral-color(lightest)
+    )
```

Comment:
I think all of these changes are being pulled in from an old merge you did.  It's reverting changes that have been made and this entire file probably needs to be reverted.